### PR TITLE
DOCS-8111 and DOCS-5374

### PR DESCRIPTION
**Pull request by sarathecoubian**
API analytics -> API analytics metrics
and API analytics retention rate + note for API Manager retention rate.

### CODE REVIEW

File: modules/ROOT/pages/performance-and-impact.adoc [hunk 1]
```diff
@@ -12,8 +12,8 @@ Anypoint Monitoring asynchronously collects data, which does not block Mule runt
 
 * Minute Resolution for Metrics.
 * Gold and Platinum subscriptions:
- ** API analytics are retained for 90 days.
- ** Application metrics are retained for 30 days. 
+ ** API analytics metrics are retained for 90 days.
```
Comment (rddheath) [inline modules/ROOT/pages/performance-and-impact.adoc:15]: Should be "metrics data", as in "data about the metrics".

File: modules/ROOT/pages/performance-and-impact.adoc [hunk 2]
```diff
@@ -12,8 +12,8 @@ Anypoint Monitoring asynchronously collects data, which does not block Mule runt
...
+ ** Application metrics are retained for 30 days.
```
Comment (rddheath) [inline modules/ROOT/pages/performance-and-impact.adoc:16]: Again, "metrics data".

File: modules/ROOT/pages/performance-and-impact.adoc (hunk 1 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/performance-and-impact.adoc:15]:
```suggestion
 ** API analytics metrics data are retained for 90 days.
```

File: modules/ROOT/pages/performance-and-impact.adoc (hunk 2 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/performance-and-impact.adoc:16]:
```suggestion
 ** Application metrics data are retained for 30 days.
```

**Pull request merged**